cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/Executor.java

```diff
@@ -1374,8 +1380,20 @@ public void run() {
         if (_executionTimerInvolveBrokerRemovalOrDemotion != null) {
           _executionTimerInvolveBrokerRemovalOrDemotion.update(duration, TimeUnit.MILLISECONDS);
         }
+
+        String executionStatusString = String.format(" task Id: %s; removed brokers: %s; demoted brokers: %s; total time used: %dms.",
```

Comment:
nit: you don't need the leading space since both log messages have one (alternatively, nuke those and keep this one)

Reply:
Good catch! Updated.